package.json

```diff
@@ -33,6 +33,7 @@
     "@fontsource/roboto": "^4.5.1",
     "@polymer/iron-component-page": "^4.0.1",
     "@rollup/plugin-terser": "^0.4.4",
+    "@types/mocha": "^10.0.7",
```

Comment:
Consider adding this in a separate PR, I believe existing tests e.g. for `button` would benefit from this.

Reply:
There's a separate PR for this in https://github.com/vaadin/web-components/pull/7648